LFS171x/fabric-material/tuna-app/src/recordTuna.js

```diff
@@ -1,10 +1,12 @@
 'use strict';
 /*
 * SPDX-License-Identifier: Apache-2.0
-*/
-/*
+
  * Chaincode Invoke
- */
+ 
+Parts of this code was contributed by the Hyperledger Team 
```

Comment:
Hyperledger Team --> Hyperledger Fabric community

In addition, instead of contributed, we should use same phrasing that you used in the README. Something like "This code is based on code written by the Hyperledger Fabric community."

Reply:
done